prepare_sector_split.R

```diff
@@ -79,7 +79,8 @@ company_ids_primary_energy_split <- readr::read_csv(
 if (config_project_parameters$remove_inactive_companies) {
   abcd_removed_inactive_companies <- readr::read_csv(
     file.path(config_dir$dir_abcd, "abcd_removed_inactive_companies.csv"),
-    col_select = cols_abcd
+    col_select = cols_abcd,
```

Comment:
I still get this warning (the first time run in a session)
```r
Warning message:
Using an external vector in selections was deprecated in tidyselect 1.1.0.
ℹ Please use `all_of()` or `any_of()` instead.
  # Was:
  data %>% select(cols_abcd)

  # Now:
  data %>% select(all_of(cols_abcd))

See <https://tidyselect.r-lib.org/reference/faq-external-vector.html>.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 
```

suggest using `dplyr::all_of()`
```suggestion
    col_select = dplyr::all_of(cols_abcd),
```

Reply:
makes sense